src/orga/minutes/2020-09-02.md

```diff
@@ -0,0 +1,178 @@
+    * Add OVH to this yaml: https://github.com/conda-forge/conda-forge.github.io/blob/master/src/inst_partners.yaml
+    * Render that yaml into jinja2
+
+* (SC) R bindings for mamba: https://github.com/marimeireles/rhumba
+    * Screencast: https://user-content.gitter-static.net/eb15dbe3c06bffa8af391f8f58f90240efd15cd5/68747470733a2f2f757365722d696d616765732e67697468756275736572636f6e74656e742e636f6d2f31373630303938322f38393636393235362d61376536626130302d643863652d313165612d383637362d3333313130386532386561322e676966
+
+* (MVN) 
+    * [mapping detector](https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/pypi_name_mapping.py)
+    * [mappings](https://github.com/regro/cf-graph-countyfair/tree/master/mappings/pypi)
+    * Mappings from conda <-> PyPi <-> import_name
+
+### Active votes
+
+### Subteam updates
+
+#### Bot
+
+#### ARM
+
+#### POWER
+
+#### CUDA
+
+#### Docs
+
+#### staged-recipes
+
+#### website
+
+#### security+systems
+
+### CI infrastructure
+
+#### Compiler upgrade
+
+### CFEP updates
+
+#### Open PRs
+
+* [cfep-04](https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/7) X11 and CDT policy
+    * INACTIVE - Merge in with some inactive-esque status?
+    * Needs new champion. Thanks for your work on this pkgw! Has unaddressed comments from pkgw as from Jan 10, 2020
+
+* [cfep-06](https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/9) Staged-recipes review lifecycle
+    * INACTIVE - Merge in with some inactive-esque status?
+    * Lingering comment from @saraedum. @jakirkham, can you reply? Has unadressed comment from @saraedum from Jan 8, 2020
+    * (MRB) The stalebot has solved the worst of the issues here. I think we could defer this one permanently.
+
+* [cfep-10](https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/15) Feedstock statuses, unmaintained
+    * INACTIVE - Merge in with some inactive-esque status?
+    * Needs another review. Has unaddressed updates from pkgw as of Jan 11, 2020
+
+* [cfep-12](https://github.com/conda-forge/cfep/pull/23) Removing packages that violate the terms of the source package
+    * Stalled since May 26, 2020
+    * Active debate about moving to "broken" vs deleting from conda-forge channel
+    * Active vote, ends on 2020-03-11
+    * What were the results of the vote?
+    * Did we hear back from NumFOCUS?
+
+* [cfep-17](https://github.com/conda-forge/cfep/pull/32) Handling pin backports and dependency rebuilds
+    * Stalled debate about implementation details between Isuru, CJ and Matt
+    * **UPDATE 2020-07-22**: We in principle have agreement to render the extra pinnings needed directly in the feedstock 
+      on a temporary basis (i.e., until the migration has ended).
+
+## Discussion
+
+## Check in on previous action items
+Copy previous action items from last meeting agenda.
+
+### This meeting
+* [ ] (ED) Update governance docs with similar voting model as what got put into conda-tools (+3 with no -1 is a pass)
```

Comment:
this stuff never made the final vote for that governance doc

Reply:
wait really? i thought that went in

Reply:
ah looks like it got discussed but never added to the doc?

Reply:
yup